### ISSUE TITLE
Bug: mod_mariadb connection cannot reconnect during db failover/restart

### ISSUE THREAD

**Issue by nneul** [signalwire/freeswitch#522]
If you have a DB cluster (maybe also with a single server) - the mod_mariadb reconnect code does not work. It is necessary to set the MYSQL_OPT_RECONNECT setting in mysql_options.

```
2020-03-16 19:49:54.530062 [WARNING] mod_mariadb.c:197 mysql_ping returned bad connection [Error: MySQL server has gone away]; reconnecting...
2020-03-16 19:49:54.530062 [CRIT] mod_mariadb.c:200 mariadb_reconnect returned bad connection -- reconnection failed! [Error: MySQL server has gone away]
```

(along with a flooded log message that includes database credentials... that's a separate issue I'd really like to see improved)

**Comment by nneul** [comment]
@andywolk

**Comment by andywolk** [comment]
Fixed by #523
